src/dsp_tools/commands/validate_data/validate_data.py

```diff
@@ -156,6 +167,7 @@ def _print_shacl_validation_violation_message(
             _save_unexpected_results_and_inform_user(
                 sorted_problems.unexpected_shacl_validation_components, report, filepath
             )
+    return errors_found
```

Comment:
This is counter-intuitive for 2 reasons:
 - There is no logical connection between the name `_print_shacl_validation_violation_message`  and the returned bool. Ideally, as a caller, I want to look at the name of the function, and immediately understand the meaning of the returned bool.
 - Intuitively, "true" is rather connected to "good", but here it is reversed: `errors_found=True` is negative.